src/components/Pages/EventList/EventListEntry.js

```diff
@@ -8,100 +8,127 @@ import { useMobileQuery } from 'js/helpers/reactMediaQueries.js';
 import { formatTimeLang } from 'js/helpers/cleanData';
 import { events as i18n } from 'js/i18n/definitions';
 
-const EventDateCalendar = ({date}) => {
+const EventDateCalendar = ({ date }) => {
```

Comment:
most everything is jsprettier

Reply:
We should add jsprettier to our normal dev workflow. If we only do it once every x months then it gets hard to see what has changed in a PR.